dts/bindings/vendor-prefixes.txt

```diff
@@ -719,6 +719,7 @@ vot	Vision Optical Technology Co., Ltd.
 vxt	VXT Ltd
 wand	Wandbord (Technexion)
 waveshare	Waveshare Electronics
+wch	WinChipHead
```

Comment:
The rule about this is that it should came from Linux mainline. Is there any entry there about this manufacturer?

Reply:
```
$ rg -i "WinChipHead"
Documentation/usb/usb-serial.rst
466:Winchiphead CH341 Driver
469:  This driver is for the Winchiphead CH341 USB-RS232 Converter. This chip
474:  The manufacturer's website: http://www.winchiphead.com/.

drivers/usb/serial/Kconfig
107:	tristate "USB Winchiphead CH341 Single Port Serial Driver"
109:	  Say Y here if you want to use a Winchiphead CH341 single port

drivers/usb/serial/ch341.c
7: * ch341.c implements a serial port driver for the Winchiphead CH341.
866:MODULE_DESCRIPTION("Winchiphead CH341 USB Serial driver");
```

I am not sure where the official manufacturer entries are maintained in the Linux kernel tree.

Reply:
I think https://elixir.bootlin.com/linux/v6.11.1/source/Documentation/devicetree/bindings/vendor-prefixes.yaml? Weird I thought it was a file like this one, anyway, there's none.